lib/commons/aria/index.js

```diff
@@ -10,177 +10,232 @@ var aria = (commons.aria = {}),
 lookupTable.attributes = {
 	'aria-activedescendant': {
 		type: 'idref',
-		allowEmpty: true
+		allowEmpty: true,
+		unsupported: false
 	},
 	'aria-atomic': {
 		type: 'boolean',
-		values: ['true', 'false']
+		values: ['true', 'false'],
+		unsupported: false
 	},
 	'aria-autocomplete': {
 		type: 'nmtoken',
-		values: ['inline', 'list', 'both', 'none']
+		values: ['inline', 'list', 'both', 'none'],
+		unsupported: false
 	},
 	'aria-busy': {
 		type: 'boolean',
-		values: ['true', 'false']
+		values: ['true', 'false'],
+		unsupported: false
 	},
 	'aria-checked': {
 		type: 'nmtoken',
-		values: ['true', 'false', 'mixed', 'undefined']
+		values: ['true', 'false', 'mixed', 'undefined'],
+		unsupported: false
 	},
 	'aria-colcount': {
-		type: 'int'
+		type: 'int',
+		unsupported: false
 	},
 	'aria-colindex': {
-		type: 'int'
+		type: 'int',
+		unsupported: false
 	},
 	'aria-colspan': {
-		type: 'int'
+		type: 'int',
+		unsupported: false
 	},
 	'aria-controls': {
 		type: 'idrefs',
-		allowEmpty: true
+		allowEmpty: true,
+		unsupported: false
 	},
 	'aria-current': {
 		type: 'nmtoken',
 		allowEmpty: true,
-		values: ['page', 'step', 'location', 'date', 'time', 'true', 'false']
+		values: ['page', 'step', 'location', 'date', 'time', 'true', 'false'],
+		unsupported: false
 	},
 	'aria-describedby': {
 		type: 'idrefs',
-		allowEmpty: true
+		allowEmpty: true,
+		unsupported: false
+	},
+	'aria-describedat': {
```

Comment:
I can leave it off, I only added it because it was mentioned in the aria-supported diff. But if we did include it, if someone tried to use it they'd see it was unsupported. I like the idea of an "unstandardized" flag in here.